tests/unit/test-cmd/test.lint.js

```diff
@@ -96,13 +104,16 @@ describe('lint', () => {
       boring: 'boring flag',
       // $FLOW_IGNORE: wrong type used for testing purpose
       selfHosted: 'self-hosted flag',
+      // $FLOW_IGNORE: wrong type used for testing purpose
+      warningsAsErrors: 'warnings-as-errors flag',
```

Comment:
you don't need to add this parameter to the test because the "pass through" is already covered by the first test you added. Also, this pass through test is kind of silly.